Core/Debugger/MemBlockInfo.cpp

```diff
@@ -207,7 +207,13 @@ void MemSlabMap::Slab::DoState(PointerWrap &p) {
 	Do(p, ticks);
 	Do(p, pc);
 	Do(p, allocated);
-	Do(p, tag);
+	if (s >= 2) {
+		Do(p, tag);
+	} else {
+		std::string stringTag;
```

Comment:
should we not initialize to tag, so it gets written properly?

Reply:
It can't ever be writing when s < max.

-[Unknown]

Reply:
oh duh. right :)